docs/reference/run.md

```diff
@@ -1365,9 +1365,11 @@ Similarly the operator can set the **hostname** with `-h`.
 
 ### TMPFS (mount tmpfs filesystems)
 
-    --tmpfs=[]: Create a tmpfs mount with: container-dir[:<options>], where the options are identical to the Linux `mount -t tmpfs -o` command.
+    --tmpfs=[]: Create a tmpfs mount with: container-dir[:<options>], where the
+    options are identical to the Linux `mount -t tmpfs -o` command.
```

Comment:
Yeah, it's probably the highlight.js detecting the wrong language, causing `--` to be treated as a "comment" :disappointed:. I can indent the second line though; basically, I wrapped to prevent a horizontal scrollbar